src/application/index.js

```diff
@@ -254,32 +246,29 @@ export default class ModV {
     }
   }
 
-  loop(delta) {
-    const {
-      meyda: { features: featuresToGet }
-    } = this.store.state;
-
-    const features = this.meyda.get(featuresToGet);
-    if (features) {
-      this.updateBeatDetektor(delta, features);
-      features.byteFrequencyData = Array.from(getByteFrequencyData() || []);
```

Comment:
Where was this moved? Because for ISF we actually added `features.byteTimeDomainData = Array.from(getByteTimeDomainData() || []);`. I wanted to add this, but couldn't find the correct position in this PR.

Reply:
@TimPietrusky ah yes, I knew there was a reason I kept this as a draft.

We can't use the WebAudio API to extract the frequency data anymore. Meyda uses an FFT library internally so we should see how they analyse the frame of audio to get the same data as we had before.

Reply:
@TimPietrusky though in the case of the time domain data (that wasn't highlighted, it's actually in the conflict) we can just use the buffer from Meyda, as we do in Waveform. 

https://github.com/vcync/modV/blob/ca0a0c6eed8e7f70441e3cf0be02f1848ce47417/src/application/sample-modules/Waveform.js#L84-L113

Reply:
@TimPietrusky should be fixed in the latest commit.